cypress/integration/chi-custom-elements/drawer.spec.js

```diff
@@ -126,4 +126,14 @@ describe('Drawer', function() {
       .should('be.visible');
 
   });
+
+  it('Should close when the user clicked outside of Drawer. ', function() {
+    cy.get('.test-close-on-outside-click .-text--h2')
```

Comment:
It's better to always rely on [data-cy] tags for getting some element since they should not change during development.